robottelo/config/base.py

```diff
@@ -514,6 +514,43 @@ def validate(self):
         return validation_errors
 
 
+class GCESettings(FeatureSettings):
+    """Google Compute Engine settings definitions."""
+
+    def __init__(self, *args, **kwargs):
+        super(GCESettings, self).__init__(*args, **kwargs)
+        self.project_id = None
+        self.client_email = None
+        self.cert_path = None
+        self.zone = None
+
+    def read(self, reader):
+        """Read GCE settings."""
+        self.project_id = reader.get('gce', 'project_id')
+        self.client_email = reader.get('gce', 'client_email')
+        self.cert_path = reader.get('gce', 'cert_path')
+        self.zone = reader.get('gce', 'zone')
+
+    def validate(self):
+        """Validate GCE settings."""
+        valid_cert_path = '/usr/share/foreman/'
+        valid_zones = [
+            'us-east1-b', 'us-east1-c', 'us-east1-d',
+            'us-east4-a', 'us-east4-b', 'us-east4-c']
+        validation_errors = []
+        if not all(self.__dict__.values()):
+            validation_errors.append(
+                'All [gce] {} options must be provided'.format(self.__dict__.keys()))
+        if not str(self.cert_path).startswith(valid_cert_path):
+            validation_errors.append(
+                '[gce] cert_path - cert should be available '
+                'from satellites {}'.format(valid_cert_path))
+        if str(self.zone).startswith('us') and self.zone not in valid_zones:
+            validation_errors.append(
+                '[gce] zone - The us zone should be one of {}'.format(valid_zones))
```

Comment:
no need to characterize to be 'us' zone. 
```suggestion
                '[gce] zone - The zone should be one of {}'.format(valid_zones))
```

Reply:
Because we are mostly tend to use us zones(and all the GCE tests will use us zones as our jenkins is there) and we have if condition for us only.